pandas/core/indexes/range.py

```diff
@@ -673,6 +681,14 @@ def __floordiv__(self, other):
                     start, start + 1, 1, name=self.name)
         return self._int64index // other
 
+    def all(self):
+        if 0 in self._range:
+            return False
+        return True
```

Comment:
this can be reduced to `return 0 not in self._range`

Reply:
Yeah, that's better. Changed.